lasagne/layers/embedding.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
+import theano.tensor as T
+
+from .. import init
+from .base import Layer
+
+
+__all__ = [
+    "EmbeddingLayer"
+]
+
+
+class EmbeddingLayer(Layer):
+    """
+    A layer for word embeddings. The input should be a `theano.Tensor.ivector``
+    specifying which embeddings to use.
+
+    Parameters
+    ----------
+    incoming : a :class:`Layer` instance or a tuple
+        The layer feeding into this layer, or the expected input shape
+
+    input_size: int
+        The Number of different embeddings. The last embedding will have index
+        input_size - 1.
+
+    output_size : int
+        The size of each embedding.
+
+    W : Theano shared variable, numpy array or callable
+        The embedding matrix.
+
+    Examples
+    --------
+    >>> from lasagne.layers import EmbeddingLayer, InputLayer, helper
```

Comment:
I'd directly import `get_output` instead of `helper` (it's already available directly in `lasagne.layers`). Just for consistency across the library.